static/.eslintrc

```diff
@@ -41,18 +41,39 @@ Do not edit this file in your local project
     "comma-dangle": [2, "never"],
     "comma-spacing": [2, { "before": false, "after": true }],
     "comma-style": [2, "last"],
+    "computed-property-spacing": [2, "never"],
```

Comment:
It depends whether we really want consistent style or not. `even` only enforces you put N spaces after `[` and before `]`. N can be 0, 1, ...

I'd rather have a strict rule saying `foo[bar]` so N = 0.